### PR TITLE
Docs: Update howto/notebook + READMEs

### PR DESCRIPTION
**Pull request by jprochazk**
### What

- Closes https://github.com/rerun-io/rerun/issues/6776

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6782?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6782?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6782)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: docs/content/howto/notebook.md [hunk 1]
```diff
@@ -121,6 +121,33 @@ rrb.Vertical(
...
+
+import numpy as np
+import rerun as rr
+from rerun.utilities import build_color_grid
+
+rr.init("rerun_example_notebook")
+rr.notebook_show()
+
+STEPS = 100
+twists = math.pi * np.sin(np.linspace(0, math.tau, STEPS)) / 4
```
Comment (emilk) [inline docs/content/howto/notebook.md:143]:
```suggestion
twists = math.tau * np.sin(np.linspace(0, math.tau, STEPS)) / 8
```

**Comment by jprochazk** [comment]
Rendered: https://landing-63ot6tefb-rerun.vercel.app/docs/howto/notebook

**Review by emilk — APPROVED** [review]
Great!

File: docs/content/howto/notebook.md (hunk 1 above)
Comment (nikolausWest) [reply docs/content/howto/notebook.md:143]: I think we should prefer pi over tau since it's more accessible. (Fewer people know about tau)

**Comment by nikolausWest** [comment]
Do we still use iFrames?
Screenshot 2024-07-05 at 12 03 40

**Pull request merged**